pkg/policies/branch/branch.go

```diff
@@ -356,6 +369,20 @@ func check(ctx context.Context, rep repositories, c *github.Client, owner,
 				pass = false
 			}
 		}
+
+		sp, _, err := rep.GetSignaturesProtectedBranch(ctx, owner, repo, b)
+		if sp != nil {
```

Comment:
use the sp.GetEnabled accessor https://pkg.go.dev/github.com/google/go-github/v45/github#SignaturesProtectedBranch.GetEnabled